### PR TITLE
newlines help with copy-paste

### PR DESCRIPTION
**Pull request by mcfnord**
This PR should result in commands appearing on their own lines, which eases the copy-paste interaction.

### CODE REVIEW

File: wiki/en/Running-a-Server.md [hunk 1]
```diff
@@ -58,9 +58,18 @@ Most people run Jamulus on a 3rd party/cloud host as a "headless" Server (no vid
 
 To run a headless server on Linux, the following steps assume you are familiar with the command line and Debian/Ubuntu or similar distribution that uses systemd.
 
-1. Download the setup script: `curl https://raw.githubusercontent.com/jamulussoftware/jamulus/main/linux/setup_repo.sh > setup_repo.sh`
-1. Make the script executable: `chmod +x setup_repo.sh`
-1. Run the script and install the headless server: `sudo ./setup_repo.sh && sudo apt install jamulus-headless`
+1. Download the setup script:
+
+	`curl https://raw.githubusercontent.com/jamulussoftware/jamulus/main/linux/setup_repo.sh > setup_repo.sh`
```
Comment (pljones) [inline wiki/en/Running-a-Server.md]: For a block of code, use three backticks:
````
```
block of code
```
````
You should also get the "Copy" button, then: image

Comment (mcfnord) [reply wiki/en/Running-a-Server.md]: oh that's how!

**Review by pljones — APPROVED** [review]
https://github.com/jamulussoftware/jamuluswebsite/blob/e915f463fcd1e068a6883b6f9e2f03b5722c86e5/wiki/en/Running-a-Server.md

Looks good.

File: wiki/en/Running-a-Server.md (hunk 1 above)
Comment (ann0see) [reply wiki/en/Running-a-Server.md]: I'd rather like spaces tbh. I don't think we have tabs anywhere else.

Comment (mcfnord) [reply wiki/en/Running-a-Server.md]: I also prefer 4 spaces but accept in-file precedent, too.

**Review by ann0see — APPROVED** [review]

**Pull request merged**